specification/repository.md

```diff
@@ -144,3 +144,9 @@ approval is granted, GDI repositories MUST NOT cut a GA release.
 - MUST document how to configure log correlation
 - MUST define minimum supported version of each auto-instrumentation framework
   - SHOULD define maximum supported version of each auto-instrumentation framework
+
+## Real User Monitoring Libraries
+
+- MUST document all supported configuration parameters and how to set them
+- MUST document how to configure manual instrumentation
+- MUST document supported instrumentations
```

Comment:
```suggestion
- MUST document supported instrumentations
- MUST document supported platforms
```

Reply:
what does "platforms" mean in this context?

Reply:
`os.type` ?

Reply:
Yeah, perhaps wrong term -- what versions of which browsers? what version of android/iOS -- any limitations from a device perspective?

Reply:
Got it. Yep, documenting, for example, that android RUM is only supported on API level 21 and up, and requires core library desugaring for API levels 21-24 is definitely very very important.